dev/RELEASE_PROCESS.rst

```diff
@@ -3,25 +3,65 @@ Release Process
+   ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27mu-manylinux1_x86_64.whl``
+   where ``<hash>`` is replaced by the SHA of the commit and the ``<version>``
+   is the incremented version from the previous step.
+
+3. **Create a release branch:** This branch should also have the same SHA as the
+   previous two steps. In order to create the branch, locally checkout the SHA
+   i.e. ``git checkout <hash>``. Then checkout a new branch of the format
+   ``releases/<release_number>``. The release number must match the increment in
+   the first step. Then push that branch to the ray repo:
+   ``git push upstream releases/<release_number>``.
 
-2. **Testing:** Before a release is created, significant testing should be done.
+4. **Testing:** Before a release is created, significant testing should be done.
    Run the script `ci/stress_tests/run_stress_tests.sh`_ and
    `ci/stress_tests/run_application_stress_tests.sh`_ and make sure it
-   passes. *And make sure it is testing the right version of Ray!* This will use
-   the autoscaler to start a bunch of machines and run some tests. Any new
-   stress tests should be added to this script so that they will be run
+   passes. *And make sure it is testing the wheels created in step 2!* This
+   will use the autoscaler to start a bunch of machines and run some tests.
+   Any new stress tests should be added to this script so that they will be run
    automatically for future release testing.
 
 3. **Libraries:** Make sure that the libraries (e.g., RLlib, Tune, SGD) are in a
    releasable state. TODO(rkn): These libraries should be tested automatically
    by the script above, but they aren't yet.
 
-4. **Increment the Python version:** Create a PR that increments the Python
-   package version. See `this example`_.
-
-5. **Create a GitHub release:** Create a GitHub release through the `GitHub
+5. **Resolve release-blockers:** Should any release blocking issues arise,
+   there are two ways these issues are resolved: A PR to patch the issue or a
+   revert commit that removes the breaking change from the release. In the case
+   of a PR, that PR should be created against master. Once it is merged, the
+   release master should ``git cherry-pick`` the commit to the release branch.
+   If the decision is to revert a commit that caused the release blocker, the
+   release master should ``git revert`` the commit to be reverted on the
+   release branch. Push these changes directly to the release branch.
+
+6. **Download all the wheels:** Now the release is ready to begin final
+   testing. The wheels are automatically uploaded to S3, even on the release
+   branch. The wheels can be found at the follow URLS Where ``<hash>`` is 
+   the SHA from the release branch most recent commit and ``<version>`` 
+   is the version incremented in Part 1:
+
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27mu-manylinux1_x86_64.whl``
```

Comment:
How about make this a runnable script like

```
export RAY_HASH=...  # e.g., 618147f57fb40368448da3b2fb4fd213828fa12b
export RAY_VERSION=...  # e.g., 0.6.6
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp27-cp27mu-manylinux1_x86_64.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp35-cp35m-manylinux1_x86_64.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp36-cp36m-manylinux1_x86_64.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37m-manylinux1_x86_64.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp27-cp27m-macosx_10_6_intel.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp35-cp35m-macosx_10_6_intel.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp36-cp36m-macosx_10_6_intel.whl
wget https://s3-us-west-2.amazonaws.com/ray-wheels/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37m-macosx_10_6_intel.whl
```

Reply:
Maybe instead of `wget`, you just want to echo the lines actually or echo `pip install -U ...`